core/common-util/src/main/java/uk/gov/gchq/gaffer/commonutil/CollectionUtil.java

```diff
@@ -46,4 +47,36 @@ private CollectionUtil() {
 
         return treeSet;
     }
+
+    public static boolean containsAny(final Collection collection, final Object[] objects) {
```

Comment:
That sounds reasonable - I just wanted to make sure we'd considered use of the libraries.